src/components/form-container.stories.jsx

```diff
@@ -0,0 +1,36 @@
+import Form from "./form";
```

Comment:
Let's enforce a standard of always using named exports instead of default exports. The benefit is that your components will always be named consistently when used throughout your codebase (unless specifically renamed to prevent name collisions or something).

```suggestion
import { Form } from "./form";
```